src/screens/FeedsManager/FeedsManagerScreen.tsx

```diff
@@ -1,24 +1,84 @@
 import React from 'react'
 
+import { gql, useMutation } from '@apollo/client'
+import { notifyErrorMsg, notifySuccessMsg } from 'actionCreators'
+import { useDispatch } from 'react-redux'
 import { Redirect, useParams } from 'react-router-dom'
 import { GraphqlErrorHandler } from 'src/components/ErrorHandler/GraphqlErrorHandler'
 import { Loading } from 'src/components/Feedback/Loading'
 import { useFeedsManagerWithProposalsQuery } from 'src/hooks/queries/useFeedsManagerWithProposalsQuery'
+import { useMutationErrorHandler } from 'src/hooks/useMutationErrorHandler'
 import NotFound from 'src/pages/NotFound'
 import { FeedsManagerView } from './FeedsManagerView'
 
+export const ENABLE_FEEDS_MANAGER_MUTATION = gql`
+  mutation EnableFeedsManager($id: ID!) {
+    enableFeedsManager(id: $id) {
+      ... on EnableFeedsManagerSuccess {
+        feedsManager {
+          id
+          name
+          uri
+          publicKey
+          isConnectionActive
+          createdAt
+          disabledAt
+        }
+      }
+      ... on NotFoundError {
+        message
+        code
+      }
+    }
+  }
+`
+
+export const DISABLE_FEEDS_MANAGER_MUTATION = gql`
+  mutation DisableFeedsManager($id: ID!) {
+    disableFeedsManager(id: $id) {
+      ... on DisableFeedsManagerSuccess {
+        feedsManager {
+          id
+          name
+          uri
+          publicKey
+          isConnectionActive
+          createdAt
+          disabledAt
+        }
+      }
+      ... on NotFoundError {
+        message
+        code
+      }
+    }
+  }
+`
+
 interface RouteParams {
   id: string
 }
 
 export const FeedsManagerScreen: React.FC = () => {
   const { id } = useParams<RouteParams>()
+  const { handleMutationError } = useMutationErrorHandler()
+  const dispatch = useDispatch()
 
   const { data, loading, error } = useFeedsManagerWithProposalsQuery({
     variables: { id },
     fetchPolicy: 'cache-and-network',
   })
 
+  const [enableFeedsManager] = useMutation<
```

Comment:
Interesting how come we dont use this ` const [enableFeedsManager, { data, loading, error }] = useMutation(...);`?

Then we can also use the <Loading /> with the loading right?

Reply:
I followed the model from [EditFeedsManagerScreen](https://github.com/smartcontractkit/operator-ui/blob/main/src/screens/EditFeedsManager/EditFeedsManagerScreen.tsx)
But with the loading would be better I think

Reply:
ah interesting, looks good then haha